app/repository/pg/user.go

```diff
@@ -49,3 +49,30 @@ func (r Repo) Add(user entities.User) (string, error) {
 
 	return id, nil
 }
+
+func (r Repo) IsUserExists(user entities.UserID) (bool, error) {
+	var err error
+	var id string
+
+	const sqlStatement = `SELECT id FROM "user" WHERE id = $1 AND deleted_at IS NULL`
+
+	if err = r.DB.QueryRow(sqlStatement, user.ID).Scan(&id); err != nil {
```

Comment:
Why don't you want to do `err :=` here and avoid err var declaration above?

Reply:
Done